src/test/resources/s3-valid-outbound-cases.xml

```diff
@@ -68,6 +68,21 @@
 		</int-aws:request-handler-advice-chain>
 	</int-aws:s3-outbound-channel-adapter>
 
+	<int-aws:s3-outbound-channel-adapter
+			id="withChannelAdapterAttributes"
+			channel="input"
+			bucket="TestBucket"
+			accessKey="dummy"
+			secretKey="dummy"
+			remote-directory="/"
+			charset="US-ASCII"
+			phase="100"
+			auto-startup="true"
```

Comment:
The `auto-startup` is `true` by default. So, to be sure that everything works well we should use `false` here :smile: